node/process.ts

```diff
@@ -267,10 +265,40 @@ export function kill(pid: number, sig: Deno.Signal | number = "SIGTERM") {
   return true;
 }
 
+function uncaughtExceptionHandler(err, origin) {
```

Comment:
Add note about the caveat with "origin" being "unhandledRejection" for synchronous errors in top-level scope.